### ISSUE TITLE
Chicago-author-date (17th) and publishers for aritlce-journal

### ISSUE THREAD

**Issue by mgr34** [citation-style-language/styles#3386]
In most style manuals the `publisher` and `publisher-place`  is to not be included in the reference list entry. The 17th edition of Chicago-author-date is no exception [[0](http://www.chicagomanualofstyle.org/tools_citationguide/citation-guide-2.html#cg-journal)]. However, if those fields are included in your citation chicago-author-date.csl [[1](https://github.com/citation-style-language/styles/blob/master/chicago-author-date.csl)] will display it. This may not be an issue per se. I suppose I can exclude the information from the reference I am feeding into the transformation. But the upstream source contains all this information. In the long run I would like to not add an additional preprocessing step if I can avoid it. 

Is this a bug or should I just not store this additional information with this reference if I am choosing to use Chicago author date?

My thinking is that in the future if another CSL style might be chosen that happens to request that information it is available. Rendering this a valid bug.

Here is an example of the input I am feeding and the output I am recieving

**input**
```javascript
 {                                                                                                   
    "id": "lancy2007",                          
     "abstract": "In this article, I highlight contrasting perspectives in the study of mother–child play. One contrast emerges as we use the lens offered by anthropology as opposed to the more commonly used lens of psychology. A second contrast is apparent from descriptions of childhood in the ethnographic record compared to observations of children in the upper strata of modern society. Psychologists and advocates who adopt their perspective view mother–child play—from infancy—as both necessary for normal development and an unlimited good. Its self‐evident value should be impressed on those who are unenlightened. Anthropologists frequently note the absence of mother–child play and, equally important, provide culturally nuanced explanations for why this is so. Psychologists see mother–child play as natural; anthropologists see it as cultural. I conclude by questioning the wholesale exportation of a culture‐specific child‐rearing strategy that may be quite incongruent with native belief and practice.",                                                     
     "issued": { "date-parts": [[2007]]},                                                              
     "type": "article-journal",                                                                        
       "language": "English",                                                                            
      "author": [                                                                                       
        {                                                                                               
          "family": "Lancy",                                                                            
           "given": "David F."                                                                           
         }                                                                                               
       ],                                                                                                
       "container-title":"American Anthropologist",                                                      
       "issue": "2",                                                                                     
       "volume": "109",                                                                                  
       "page": "273-284",                                                                                
       "title": "Accounting for Variability in Mother–Child Play",                                       
       "publisher-place": "Berkeley, Calif",                                                             
       "publisher": "University of California Press",                                                    
       "DOI": "10.1525/aa.2007.109.2.273"                                                                
     },   

```
**output**

Lancy, David F. 2007. “Accounting for Variability in Mother–Child Play.”
*American Anthropologist* 109 (2). Berkeley, Calif: University of California
Press: 273–84. https://doi.org/10.1525/aa.2007.109.2.273.

 

[0] http://www.chicagomanualofstyle.org/tools_citationguide/citation-guide-2.html#cg-journal
[1] https://github.com/citation-style-language/styles/blob/master/chicago-author-date.csl

**Comment by adam3smith** [comment]
For a longterm we were buidling styles under the assumption that journal articles simply don't have/need that information in the metadata (based mainly on the Zotero data model but also all major citation styles), so almost all CSL styles are coded under that assumption.

It turns out, unfortunately, that this isn't universally true, so it'd be good to fix it, but that's a pretty significant undertaking with >1000 styles. That said, we'd happy take pull requests to fix this for individual styles.

**Comment by adam3smith** [comment]
closed via #3455
